extensions/strategies/trendline/strategy.js

```diff
@@ -1,113 +1,66 @@
-let math = require('mathjs')
-  , trend = require('trend')
-  , z = require('zero-fill')
-  , n = require('numbro')
+var z = require('zero-fill')
   , stats = require('stats-lite')
+  , math = require('mathjs')
   , ema = require('../../../lib/ema')
   , Phenotypes = require('../../../lib/phenotype')
-var oldgrowth = 1
+  , regression = require('regression')
+  , trend = require('trend')
+  , n = require('numbro')
 
 module.exports = {
   name: 'trendline',
-  description: 'Calculate a trendline and trade when trend is positive vs negative.',
-
+  description: 'Buy on positive trendline above 1.1 Make sure to set --days when backtesting period.',
```

Comment:
here i dont see, why its selling above 1.1? maybe i dont get it, but if i am right it will buy/sell if its above or below trendline; so the description is wrong, right?